src/Components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { Filters } from '../../types/Filters';
+import { DispatchContext, StateContext } from '../../utils/GlobalContext';
+
+export const TodoFilter: React.FC = () => {
+  const { todos, filteredTodos, filter } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+  const uncmpltTodosCount = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <>
+      {todos.length > 0 && (
```

Comment:
makes sense to add if and return null in this case
And here get rid of check and React.fragment